recipe/install_llvm.bat

```diff
@@ -9,14 +9,35 @@ if "%PKG_NAME%" == "libllvm-c%PKG_VERSION:~0,2%" (
     REM only libLLVM-C
     move .\temp_prefix\bin\LLVM-C.dll %LIBRARY_BIN%
     move .\temp_prefix\lib\LLVM-C.lib %LIBRARY_LIB%
+) else if "%PKG_NAME%" == "llvm-tools-%PKG_VERSION:~0,2%" (
+    cmake --install ./build --prefix=./temp_prefix
+    if %ERRORLEVEL% neq 0 exit 1
+
+    REM all the executables (not .dll's) in \bin with a version suffix,
+    REM except one binary that belongs to llvmdev
+    del .\temp_prefix\bin\llvm-config.exe
+
+    pushd .\temp_prefix\bin
+    for %%f in (*.exe) do (
+        echo %%~nf
+        copy "%%~nf.exe" %LIBRARY_BIN%\%%~nf-%PKG_VERSION:~0,2%.exe
+    )
+    popd
 ) else if "%PKG_NAME%" == "llvm-tools" (
     cmake --install ./build --prefix=./temp_prefix
     if %ERRORLEVEL% neq 0 exit 1
 
     mkdir %LIBRARY_PREFIX%\share
     REM all the executables (not .dll's) in \bin & everything in \share
-    move .\temp_prefix\bin\*.exe %LIBRARY_BIN%
     move .\temp_prefix\share\* %LIBRARY_PREFIX%\share
+
+    REM create wrappers (can't do symlinks on windows by default)
+    pushd .\temp_prefix\bin
+    for %%f in (*.exe) do (
+        echo call %LIBRARY_BIN%\%%~nf-%PKG_VERSION:~0,2%.exe %%* > %LIBRARY_BIN%\%%~nf.exe
```

Comment:
This is bad. With this change, if you do
```cmd
llvm-as --version
echo "hello"
```
the second command won't print and you have to d
```cmd
call llvm-as --version
echo hello
```

Reply:
Hm, that's a pity. I had seen the [wrappers](https://github.com/conda-forge/gradle-feedstock/blob/abbfb81f25370a0e00ad7f67a0e2136ab023f119/recipe/bld.bat#L13-L15) on the gradle feedstock, thought I'd give it a shot here.

Didn't know that this doesn't work for regular usage, thanks for the info. I presume there's no better way to wrap? Or couldn't we compile our own wrapper-exe (per binary) that just calls the versioned one from C/C++?